resources/assets/js/app.js

```diff
@@ -6,17 +6,3 @@
  */
 
 require('./bootstrap');
-
```

Comment:
Assuming this is how you want it - and don't need Vue.

Reply:
Nope, we don't use it.